java/light-4j/pom.xml

```diff
@@ -0,0 +1,428 @@
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>benchmark.light-4j</groupId>
+    <artifactId>benchmark</artifactId>
+    <packaging>jar</packaging>
+    <name>benchmark</name>
+    <version>0.0.1</version>
+
+    <properties>
+        <java.version>1.8</java.version>
+        
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.main.class>com.networknt.server.Server</project.main.class>
+        <version.light-4j>1.6.23</version.light-4j>
+        <version.jackson>2.10.0</version.jackson>
```

Comment:
is there any restriction to use `*` ? I mean it could better to have results with last version, if stable, than having to ping each dependencies manually

Reply:
At least for Jackson, when I tried using version 3, the application failed to run due to a missing method. I did update framework versions to allow for newer versions.

Reply:
perhaps `[2,3)` could work

Reply:
Okay, I managed to get it working with a range in 18bddd3079846cb9e0607fbf32d5045a33d70b1a.